tests/fuzz/fuzz-bech32.c

```diff
@@ -60,5 +60,18 @@ void run(const uint8_t *data, size_t size)
 		assert(memcmp(data_out, data, data_out_len) == 0);
 	}
 
+	/* Test 8-to-5 bit roundtrip conversion */
```

Comment:
Would it make sense to do the roundtrip as part of the above call to `bech32_convert_bits`?